src/Toggle/index.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+import styled from 'styled-components'
+
+export interface ToggleProps {
+  width?: string
+  height?: string
+  borderRadius?: string
+  padding?: number
+  offBackground?: string
+  checkedBackground?: string
+  buttonColor?: string
+  checked?: boolean
+  onChange?: (e: React.ChangeEvent<HTMLInputElement>) => void
```

Comment:
any reason why they are optional?

Reply:
Given it's optional on normal input controls does this make sense for consistency?

Reply:
the toggle still works without setting `checked` or providing an `onChange` function

Reply:
I could add defaults for those props